cmd/login_pat.go

```
@@ -164,6 +168,67 @@ func promptLoginURL(asJSON bool) string {
+// with more than one. A bare global login keeps "default" (the documented
+// convention); returns "" for that and for single-profile machines. It steps
+// in for --local, and for a --url whose host is not default's control plane:
+// both would otherwise commit a directory or overwrite default on a guess.
+//
+// On a terminal it lists the store and asks — a number, an existing name, or
+// a NEW name (which then goes through the URL and PAT prompts). Without a
+// terminal, or with --json, it exits 2 with the list and a -p hint.
+func pickProfile(out io.Writer, asJSON, local bool, flagURL string) (string, error) {
+	store, err := credentials.Load()
+	if err != nil || len(store) < 2 {
+		return "", nil
```

golangci-lint (2.13.2)

[error] error is not nil (line 181) but it returns nil

(nilerr)